### ISSUE TITLE
Run on windows via linux subsystem

### ISSUE THREAD

**Issue by bwtan** [ClearAnatomics/ClearMap#7]
Hi Christoph,

Would that be possible to run the pipeline on the windows subsystem for linux (https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux)? Will there be an support changes for this?

**Comment by ChristophKirst** [comment]
We don't have much experience running ClearMap on windows. You can try to install anaconda there or run a linux environment via docker.
We ran ClearMap on a linux via an oracle virtual machine once.
We did not test large data sets.

Hope that helps

**Comment by nicolasrenier** [comment]
We were running ClearMap on a linux virtual machine on windows a while back, but there were issues with the parallel processing, so we dropped it. After that, we used dualboot systems, but after a while it was too painful to manage so we now use dedicated linux machines. It is hard for new users to commit a machine to linux for these analysis, but if you really get into brain mapping, this ends up being the best investment.

Our hope is to hire someone in the future to help porting the system to Windows, so we'll definitely look into it, but can't promise anything at this point...
